src/prosper/feature.js

```diff
@@ -1 +1,63 @@
-console.log("Prosper")
+/** 
+Activity one (data types)
+--------------------------
+--------------------------
+what data type?
+14 ----------------------- Number
+-80 ---------------------- Number
+8.5 ---------------------- Number
+true --------------------- Boolean
+(5 < 7) ------------------ Boolean
+null --------------------- Undefined
+3.141 -------------------- Number
+"Pi" --------------------- String
+67+18 -------------------- Number 
+**/
+
+
+console.log("Prosper Oyibo");				//log my name to console
+
+convertToBinary = function(x) {
```

Comment:
It's recommended you create functions using the `var` keyword. This way, you prevent the creation of global functions.